src/components/Bootstrap/InputField/ZekBvInput.vue

```diff
@@ -154,10 +154,12 @@ export default {
   },
   methods: {
     input(event) {
-      this.$emit('input', event.target.value)
+      let value = this.type === 'number' ? parseFloat(event.target.value) : event.target.value
+      this.$emit('input', value)
     },
     change(event) {
-      this.$emit('change', event.target.value)
+      let value = this.type === 'number' ? parseFloat(event.target.value) : event.target.value
```

Comment:
If you want to make sure the type is number you should do a type case, instead of parsing. `parseFloat(x) -> Number(x)`